lib/blocs/request_manager_cubit.dart

```diff
@@ -11,17 +10,14 @@ import 'package:trufi_core/trufi_models.dart';
 
 // TODO: This is actually not a Cubit it is just a service / controller
 //  there is no state that it contains
-class RequestManagerCubit extends Cubit<void> implements RequestManager {
+class RequestManagerCubit extends Cubit<void> {
   final RequestManager _offlineRequestManager;
```

Comment:
@md-weber maybe it can be renamed to search location manager

Reply:
If it is only responsible for search yeah sure, that would make it more understandable.